src/Message/ItemDataTrait.php

```diff
@@ -23,7 +23,7 @@ public function getItemData(ItemBag $items)
                 'type' => $item->getType(),
                 'name' => $item->getName(),
                 'quantity' => $item->getQuantity(),
-                'tax_rate' => (int) ($item->getTaxRate() * 100),
+                'tax_rate' => \bcmul((string) $item->getTaxRate(),  100),
```

Comment:
Could you change (or add a case) the value in `ItemDataTestTrait` to reflect this case?